<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VKE Can-BUS Bootloader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="conn_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>114</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="find_pushButton">
       <property name="text">
        <string>Find CHAI device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="channel_label">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="speed_label">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="channel_comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="speed_comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CONNECT</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="console_groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Status console</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="upd_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Update firmware</string>
    </property>
    <widget class="QPushButton" name="upd_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="upd_progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="open_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open file</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
